src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ConfigDumpReferenceCommandTest.php

```diff
@@ -115,6 +115,17 @@ public function testDumpAtPathXml(bool $debug)
         $this->assertStringContainsString('[ERROR] The "path" option is only available for the "yaml" format.', $tester->getDisplay());
     }
 
+    #[TestWith(['yaml'])]
+    #[TestWith(['xml'])]
+    public function testDumpFrameworkBundle(string $format)
+    {
+        $tester = $this->createCommandTester(true);
+        $ret = $tester->execute(['name' => 'framework', '--format' => $format]);
+
+        $this->assertSame(0, $ret, 'Returns 0 in case of success');
+        $this->assertStringContainsString('%env(default::SYMFONY_TRUSTED_PROXIES)%', $tester->getDisplay());
+    }
```

Comment:
This integration test increases the coverage of the (Yaml|Xml)ReferenceDumper classes with the configuration of the FrameworkBundle.

It's currently failing with "[Unit Tests / Unit Tests (8.4, high-deps)](https://github.com/symfony/symfony/actions/runs/17897266203/job/50885544058?pr=61800)" job because of the bug in the current `symfony/config: 7.4@dev`. That will be solved by merging this PR.